CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - ???
```

Comment:
This version contains only an addition, there's no breaking changes, so I think `1.1.0` is more accurate

Reply:
Fixed.